packages/cli-utils/src/ts/factory.ts

```diff
@@ -49,19 +49,17 @@ export const programFactory = (params: ProgramFactoryParams): ProgramFactory =>
   const virtualMap: VirtualMap = new Map();
 
   const projectRoot = path.dirname(params.configPath);
-  const tslibPath = path.join(projectRoot, 'node_modules/typescript/lib/');
-  const system = createFSBackedSystem(vfsMap, projectRoot, ts, tslibPath);
+  const system = createFSBackedSystem(vfsMap, projectRoot, ts);
   const config = resolveConfig(params, system);
-
   for (const { filename, contents } of resolveLibs(params)) {
-    if (contents) system.writeFile(path.join(tslibPath, filename), contents);
+    if (contents) system.writeFile('/' + filename, contents);
```

Comment:
`@typescript/vfs` seems to expect that the lib files are at the virtual root folder, even if an override is passed to `createFSBackedSystem`. The former's argument for this doesn't seem to be used correctly